src/sys/socket/consts.rs

```diff
@@ -1,75 +1,253 @@
 pub use self::os::*;
 
-#[cfg(any(target_os = "linux", target_os = "android"))]
 mod os {
     use libc::{self, c_int, uint8_t};
 
     pub const AF_UNIX: c_int  = libc::AF_UNIX;
```

Comment:
It definitely should. All constants should be declared in `libc` and we should just wrap them up into nicer types. It looks like that's already done for the `AF_*` constants, but it should be done for these other ones as well.